pkg/downloader/manager.go

```diff
@@ -216,6 +216,30 @@ func (m *Manager) downloadAll(deps []*chartutil.Dependency) error {
 	fmt.Fprintf(m.Out, "Saving %d charts\n", len(deps))
 	var saveError error
 	for _, dep := range deps {
+		// No repository means the chart is in charts directory
+		if dep.Repository == "" {
```

Comment:
What are your thoughts on using a keyword like `local` (which I know is already used in v2, but technically by doing `@local`, plus very few people use the local repo). Assuming an empty value means local could lead to weird error messages for people who may have just forgot and `local` is more expressive

Reply:
Also, we should add documentation of this new feature to the help text for `helm dep`

Reply:
You raise a good point here. I'll take a deeper dive into the code and see if there's an alternative approach we should take instead.

Reply:
After further review, I think it's probably fine to accept an empty repository field as on-disk. However, I think we should inform the user that Helm is attempting to load the chart from disk. I added this in the latest commit. Does that seem fair to you?

I'm afraid of adding additional "one-off" syntax to an already unruly function that desperately needs a rework

Reply:
Sounds like a good compromise